libs/transloco/src/lib/scope-resolver.ts

```diff
@@ -8,7 +15,14 @@ type ScopeResolverParams = {
 };
 
 export class ScopeResolver {
-  constructor(private service: TranslocoService) {}
+  readonly config: TranslocoConfig;
+
+  constructor(
+    private service: TranslocoService,
+    @Inject(TRANSLOCO_CONFIG) userConfig: PartialTranslocoConfig = {},
+  ) {
+    this.config = JSON.parse(JSON.stringify(userConfig));
```

Comment:
No need for the clone, we are just reading properties.

Reply:
Removed, used the `config` from the service